drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -120,6 +120,7 @@ public void setChromeOptions(WebDriverConfiguration configuration, DesiredCapabi
         String browser = configuration.getBrowser().toLowerCase();
         if (browser.equals("chromeheadless")) {
             chromeOptions.addArguments("--headless");
+            chromeOptions.addArguments("--window-size=1200,600");
```

Comment:
The `size=1200x600` was an approximation for a near about fullscreen size. Couldn't come up with the best size(may be smaller or larger). 
`dimensions` property didn't work, when executing test-suite tests with chromeheadless, it gave the error msg `could not automatically resize browser.` 
Also, when executing drone tests after this change, removed the warning `waiting for browser to resize`.